Occasional WirePattern gc test failures on CI

The WirePattern garbage collection test is failing occasionally on CI, especially on Node.js 0.12 (Travis) and on Windows. Seems like a timing issue?

```
Component traits WirePattern when grouping requests should collect garbage every N requests:
     AssertionError: expected [ Array(3) ] to have a length of 2 but got 3
    at Context.<anonymous> (C:\projects\noflo\spec\Helpers.coffee:1524:57)
```

https://ci.appveyor.com/project/bergie/noflo/build/1.0.548#L1242

cc / @trustmaster

--- a/spec/Helpers.coffee
+++ b/spec/Helpers.coffee
@@ -1517,13 +1517,13 @@ describe 'Component traits', ->
           chai.expect(data.y).to.equal reqs[data.id].y
           done() if count is 6 # 6 complete requests processed
 
-        sendRequests reqs, 6
+        sendRequests reqs, 10
 
       it 'should collect garbage every N requests', (done) ->
-        # GC dropped 2 timed out packets, 2 should be left
-        chai.expect(Object.keys(c.groupedData)).to.have.lengthOf 2
-        chai.expect(Object.keys(c.groupedGroups)).to.have.lengthOf 2
-        chai.expect(Object.keys(c.disconnectData)).to.have.lengthOf 2
+        # GC dropped 3 timed out packets, 1 should be left
+        chai.expect(Object.keys(c.groupedData)).to.have.lengthOf 1
+        chai.expect(Object.keys(c.groupedGroups)).to.have.lengthOf 1
+        chai.expect(Object.keys(c.disconnectData)).to.have.lengthOf 1
         done()
 
       it 'should be able to drop a request explicitly', (done) ->

